### ISSUE TITLE
[Suggestion] Playstore only Package

### ISSUE THREAD

**Issue by Plamper** [FriendlyNeighborhoodShane/MinMicroG#32]
Hi, I wanted to ask if it would be possible to have a package like MinimalIAPs but without microg, so it can be installed on ROMs that have MicroG preinstalled. Similar to the Aurora Services version which works flawless on Lineage for MicroG. Or alternatively could you be so kind to provide me how to modify the package so no MicroG gets installed?

Anyway, Thanks for providing these easy installers!

**Comment by FriendlyNeighborhoodShane** [comment]
A playstore-only package has been requested for by other people too, and has been on my radar for a while now.

It's just been sort of bothering me how I'll add something like that without creating a lot of confusing UX for users 😅

Like, Most of the MinMicroG packages install various configurations of microG (or UNLP) that are mutually exclusive, and people can choose to install user apps over that as they wish.

But then there is this one package, Aurora Services, which still has "MicroG" in its name, that doesn't install microG at all! And more confusingly, it's still mutually exclusive with all the microG-including packages, so you can't e.g. install it with the minimal or UNLP packages. The playstore package behaving similarly create even more of a mess, where you wouldn't be able to use various combinations of microG, Playstore, and Aurora Services.

I've been thinking of several solutions to this issue, and the one that strikes me as the cleanest is probably creating another category of packages ("MinAddons" or something) that install individual components and are not mutually-exclusive with each other or the other Min-"MicroG" packages.

Documentation is already is kinda confusing on this, to solve this it probably needs to be improved too.

Would be interesting to hear thoughts and opinions on this.

**Comment by Plamper** [comment]
I agree. I had to read the current docs twice to figure out I could flash Aurora Services but not MinimalIAPs. As you suggested it would probably be better having a minimal MicroG package and multiple Addon packages. You could still provide an "everything" package for lazy people. What I have also seen is having to edit config files to disable certain components of the package, but I think this a worse approach than just flashing multiple packages in succession, as it just adds confusing steps to the install process.

**Comment by FriendlyNeighborhoodShane** [comment]
Took a shot at it if you're still interested:

https://github.com/FriendlyNeighborhoodShane/MinMicroG/commit/d446b07e0765b6598cf2c19b44087915662259a6

CI release:

https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases/tag/2022.08.14

**Comment by gamechanger2030** [comment]
> https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases/tag/2022.08.14

I tried to install minimal microg + playstore addon.
I flashed both zips from TWRP recovery. (Minimal first, ps second)

when i rebooted to rom, playstore app showed up and was working fine.
But when i later rebooted by adding some other magisk modules(notch and Bluetooth related), the playstore app disappeared.
I found in system apps that it converted to fakestore and is not accessible for use.

Why it is happening? Am i flashing wrong way?

**Comment by FriendlyNeighborhoodShane** [comment]
> > https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases/tag/2022.08.14
> 
> I tried to install minimal microg + playstore addon. I clean flashed both zips from TWRP recovery. (Minimal first, ps second)
> 
> when i rebooted to rom, playstore app showed up and was working fine. But when i later rebooted by adding some other magisk modules(notch and Bluetooth related), the playstore app disappeared. I found in system apps that it converted to fakestore and is not accessible for use.
> 
> Microg is working fine like nothing happened. Only playstore app goes missing.
> 
> Why it is happening? Am i flashing wrong way?

Yeah, there have been *logistical* problems with trying to make fakestore and playstore coexist. The playstore addon zip works fine by itself, but when used on top of a package that already includes fakestore, it's a game of chance which of those will appear.

The separate minimalIAP zip will be back from the next release. For now just use the one from https://github.com/FriendlyNeighborhoodShane/MinMicroG-abuse-CI/releases/tag/2022.05.13

**Comment by gamechanger2030** [comment]
Sorry for the late reply, I did some little changes by my own.

I unzipped both zips
took out patchphonesky.apk from playstore addon zip
renamed it to fakestore.apk
and replaced the original fakestore.apk in minimal zip by this.
zipped the minimal and flashed from twrp.

It worked. The playstore app is showed up normally and no fakestore now in settings.
In Microg self check, after giving permissions of spoof package, there was still no check mark on - "phonesky has correct signature". Play store was also not working as well(app was opening but stuck on home loading screen).
Rest all boxes were checked in Microg self check.

But when i gave "spoof package signature" permission to playstore app by going in settings>apps>playstore, it started working. Also now all boxes in Microg check is checked.

And everything is working fine (atleast for now).
Idk if this way will cause issue in future so i'll update to minimalIAP when it will be released.
